code/client/src/pages/Create.jsx

```diff
@@ -162,14 +165,14 @@ const Create = ({ expertMode, showRecovery }) => {
   }, [])
 
   return (
-    <>
+    <div className={v2ui ? 'wallet-creation-flow' : ''}>
```

Comment:
possible to avoid custom class?

Reply:
definitely, will address that in the newui pr